### ISSUE TITLE
Apple TV: 404 error after trying open Apple TV via STF

### ISSUE THREAD

**Issue by dhreben** [zebrunner/stf#747]
**Steps:**

1. Open STF
2. Open Apple TV

**Actual result :** 404 error after trying open Apple TV via STF
**NOTE:**  Apple TV restarted and available  on STF
                    Not repro after clone from master branch (2.4.6v)

<img width="860" alt="Screenshot 2023-05-11 at 18 42 52" src="https://github.com/zebrunner/mcloud-ios/assets/50950954/8b1a25e1-6ac3-4f1a-a107-28d3078e4965">

Logs: 
```
2023-05-11T15:30:16.525Z IMP/device:plugins:group 7599 [DBF722DB-C6BB-4F72-9828-438DE90795B7] Now owned by "admi@.com"
2023-05-11T15:30:16.526Z INF/device:plugins:group 7599 [DBF722DB-C6BB-4F72-9828-438DE90795B7] Subscribing to group channel "QONKdjuwSZaB8NM9zXsWiA=="
2023-05-11 18:30:17.592 WebDriverAgentRunner_tvOS-Runner[7199:438512] Got screenshots broadcast client connection at ::1:60058
2023-05-11 18:30:17.592 WebDriverAgentRunner_tvOS-Runner[7199:438512] Starting screenshots broadcast for the client at ::1:60058
2023-05-11T15:30:17.595Z INF/wdaClient 7599 [DBF722DB-C6BB-4F72-9828-438DE90795B7] verifying wda session status...
2023-05-11 18:30:17.605 WebDriverAgentRunner_tvOS-Runner[7199:438509] Getting the most recent active application (out of 1 total items)
2023-05-11 18:30:17.605 WebDriverAgentRunner_tvOS-Runner[7199:438509] Cannot retrieve any active applications. Assuming the system application is the active one
    t =   136.50s Find the Application 'com.apple.PineBoard'
    t =   136.50s     Requesting snapshot of accessibility hierarchy for app with pid 24448
2023-05-11T15:30:17.628Z VRB/wdaClient 7599 [DBF722DB-C6BB-4F72-9828-438DE90795B7] Request has been sent to WDA with data:  {"method":"POST","uri":"http://localhost:20041/session","body":{"capabilities":{}},"json":true}
2023-05-11T15:30:17.638Z VRB/wdaClient 7599 [DBF722DB-C6BB-4F72-9828-438DE90795B7] Request has been sent to WDA with data:  {"method":"GET","uri":"http://localhost:20041/status","json":true}
2023-05-11T15:30:17.641Z INF/wdaClient 7599 [DBF722DB-C6BB-4F72-9828-438DE90795B7] status response: {"value":{"message":"WebDriverAgent is ready to accept commands","state":"success","os":{"testmanagerdVersion":28,"name":"tvOS","sdkVersion":"16.0","version":"16.0"},"ios":{"simulatorVersion":"16.0","ip":"192.168.89.189"},"ready":true,"build":{"time":"Jan 17 2023 16:01:58","productBundleIdentifier":"com.facebook.WebDriverAgentRunner"}},"sessionId":"C00F6564-9225-4157-B38E-1DF094A886D7"}
2023-05-11T15:30:17.645Z INF/wdaClient 7599 [DBF722DB-C6BB-4F72-9828-438DE90795B7] reusing existing wda session: C00F6564-9225-4157-B38E-1DF094A886D7
2023-05-11T15:30:17.649Z INF/wdaClient 7599 [DBF722DB-C6BB-4F72-9828-438DE90795B7] getting device window size...
2023-05-11 18:30:17.660 WebDriverAgentRunner_tvOS-Runner[7199:438509] Getting the most recent active application (out of 1 total items)
2023-05-11 18:30:17.660 WebDriverAgentRunner_tvOS-Runner[7199:438509] Cannot retrieve any active applications. Assuming the system application is the active one
    t =   136.56s Find the Application 'com.apple.PineBoard'
    t =   136.56s     Requesting snapshot of accessibility hierarchy for app with pid 24448
2023-05-11T15:30:17.671Z INF/util:lifecycle 7599 [DBF722DB-C6BB-4F72-9828-438DE90795B7] Winding down for graceful exit, with optional error :  StatusCodeError: 404 - {"value":{"error":"unknown command","message":"Unhandled endpoint: /session/C00F6564-9225-4157-B38E-1DF094A886D7/orientation -- http://localhost:20041/ with parameters {\n    wildcards =     (\n        \"session/C00F6564-9225-4157-B38E-1DF094A886D7/orientation\"\n    );\n}","traceback":""},"sessionId":"C00F6564-9225-4157-B38E-1DF094A886D7"}
    at new StatusCodeError (/Users/idmcloud/tools/mcloud-ios/stf/node_modules/request-promise-core/lib/errors.js:32:15)
    at Request.plumbing.callback (/Users/idmcloud/tools/mcloud-ios/stf/node_modules/request-promise-core/lib/plumbing.js:104:33)
    at Request.RP$callback [as _callback] (/Users/idmcloud/tools/mcloud-ios/stf/node_modules/request-promise-core/lib/plumbing.js:46:31)
    at Request.self.callback (/Users/idmcloud/tools/mcloud-ios/stf/node_modules/request/request.js:185:22)
    at Request.emit (node:events:390:28)
    at Request.<anonymous> (/Users/idmcloud/tools/mcloud-ios/stf/node_modules/request/request.js:1154:10)
    at Request.emit (node:events:390:28)
    at IncomingMessage.<anonymous> (/Users/idmcloud/tools/mcloud-ios/stf/node_modules/request/request.js:1076:12)
    at Object.onceWrapper (node:events:509:28)
    at IncomingMessage.emit (node:events:402:35)
    at endReadableNT (node:internal/streams/readable:1340:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) {
  statusCode: 404,
  error: {
    value: {
      error: 'unknown command',
      message: 'Unhandled endpoint: /session/C00F6564-9225-4157-B38E-1DF094A886D7/orientation -- http://localhost:20041/ with parameters {\n' +
        '    wildcards =     (\n' +
        '        "session/C00F6564-9225-4157-B38E-1DF094A886D7/orientation"\n' +
        '    );\n' +
        '}',
      traceback: ''
    },
    sessionId: 'C00F6564-9225-4157-B38E-1DF094A886D7'
  },
  options: {
    method: 'GET',
    uri: 'http://localhost:20041/session/C00F6564-9225-4157-B38E-1DF094A886D7/orientation',
    json: true,
    callback: [Function: RP$callback],
    transform: undefined,
    simple: true,
    resolveWithFullResponse: false,
    transform2xxOnly: false
  },
  response: <ref *1> IncomingMessage {
    _readableState: ReadableState {
      objectMode: false,
      highWaterMark: 16384,
      buffer: BufferList { head: null, tail: null, length: 0 },
      length: 0,
      pipes: [],
      flowing: true,
      ended: true,
      endEmitted: true,
      reading: false,
      constructed: true,
      sync: true,
      needReadable: false,
      emittedReadable: false,
      readableListening: false,
      resumeScheduled: false,
      errorEmitted: false,
      emitClose: true,
      autoDestroy: true,
      destroyed: true,
      errored: null,
      closed: true,
      closeEmitted: true,
      defaultEncoding: 'utf8',
      awaitDrainWriters: null,
      multiAwaitDrain: false,
      readingMore: true,
      dataEmitted: true,
      decoder: null,
      encoding: null,
      [Symbol(kPaused)]: false
    },
    _events: [Object: null prototype] {
      end: [Array],
      close: [Array],
      data: [Function (anonymous)],
      error: [Function (anonymous)]
    },
    _eventsCount: 4,
    _maxListeners: undefined,
    socket: Socket {
      connecting: false,
      _hadError: false,
      _parent: null,
      _host: 'localhost',
      _readableState: [ReadableState],
      _events: [Object: null prototype],
      _eventsCount: 7,
      _maxListeners: undefined,
      _writableState: [WritableState],
      allowHalfOpen: false,
      _sockname: null,
      _pendingData: null,
      _pendingEncoding: '',
      server: null,
      _server: null,
      parser: null,
      _httpMessage: [ClientRequest],
      [Symbol(async_id_symbol)]: 130,
      [Symbol(kHandle)]: [TCP],
      [Symbol(kSetNoDelay)]: false,
      [Symbol(lastWriteQueueSize)]: 0,
      [Symbol(timeout)]: null,
      [Symbol(kBuffer)]: null,
      [Symbol(kBufferCb)]: null,
      [Symbol(kBufferGen)]: null,
      [Symbol(kCapture)]: false,
      [Symbol(kBytesRead)]: 0,
      [Symbol(kBytesWritten)]: 0,
      [Symbol(RequestTimeout)]: undefined
    },
    httpVersionMajor: 1,
    httpVersionMinor: 1,
    httpVersion: '1.1',
    complete: true,
    rawHeaders: [
      'Server',
      'WebDriverAgent/1.0',
      'Content-Type',
      'application/json;charset=UTF-8',
      'Access-Control-Allow-Origin',
      '*',
      'Date',
      'Thu, 11 May 2023 15:30:17 GMT',
      'Access-Control-Allow-Headers',
      'Content-Type, X-Requested-With',
      'Accept-Ranges',
      'bytes',
      'Content-Length',
      '385',
      'Connection',
      'close'
    ],
    rawTrailers: [],
    aborted: false,
    upgrade: false,
    url: '',
    method: null,
    statusCode: 404,
    statusMessage: 'Not Found',
    client: Socket {
      connecting: false,
      _hadError: false,
      _parent: null,
      _host: 'localhost',
      _readableState: [ReadableState],
      _events: [Object: null prototype],
      _eventsCount: 7,
      _maxListeners: undefined,
      _writableState: [WritableState],
      allowHalfOpen: false,
      _sockname: null,
      _pendingData: null,
      _pendingEncoding: '',
      server: null,
      _server: null,
      parser: null,
      _httpMessage: [ClientRequest],
      [Symbol(async_id_symbol)]: 130,
      [Symbol(kHandle)]: [TCP],
      [Symbol(kSetNoDelay)]: false,
      [Symbol(lastWriteQueueSize)]: 0,
      [Symbol(timeout)]: null,
      [Symbol(kBuffer)]: null,
      [Symbol(kBufferCb)]: null,
      [Symbol(kBufferGen)]: null,
      [Symbol(kCapture)]: false,
      [Symbol(kBytesRead)]: 0,
      [Symbol(kBytesWritten)]: 0,
      [Symbol(RequestTimeout)]: undefined
    },
    _consuming: false,
    _dumped: false,
    req: ClientRequest {
      _events: [Object: null prototype],
      _eventsCount: 5,
      _maxListeners: undefined,
      outputData: [],
      outputSize: 0,
      writable: true,
      destroyed: false,
      _last: true,
      chunkedEncoding: false,
      shouldKeepAlive: false,
      maxRequestsOnConnectionReached: false,
      _defaultKeepAlive: true,
      useChunkedEncodingByDefault: false,
      sendDate: false,
      _removedConnection: false,
      _removedContLen: false,
      _removedTE: false,
      _contentLength: 0,
      _hasBody: true,
      _trailer: '',
      finished: true,
      _headerSent: true,
      _closed: false,
      socket: [Socket],
      _header: 'GET /session/C00F6564-9225-4157-B38E-1DF094A886D7/orientation HTTP/1.1\r\n' +
        'host: localhost:20041\r\n' +
        'accept: application/json\r\n' +
        'Connection: close\r\n' +
        '\r\n',
      _keepAliveTimeout: 0,
      _onPendingData: [Function: nop],
      agent: [Agent],
      socketPath: undefined,
      method: 'GET',
      maxHeaderSize: undefined,
      insecureHTTPParser: undefined,
      path: '/session/C00F6564-9225-4157-B38E-1DF094A886D7/orientation',
      _ended: true,
      res: [Circular *1],
      aborted: false,
      timeoutCb: null,
      upgradeOrConnect: false,
      parser: null,
      maxHeadersCount: null,
      reusedSocket: false,
      host: 'localhost',
      protocol: 'http:',
      [Symbol(kCapture)]: false,
      [Symbol(kNeedDrain)]: false,
      [Symbol(corked)]: 0,
      [Symbol(kOutHeaders)]: [Object: null prototype]
    },
    request: Request {
      _events: [Object: null prototype],
      _eventsCount: 5,
      _maxListeners: undefined,
      method: 'GET',
      uri: [Url],
      readable: true,
      writable: true,
      explicitMethod: true,
      _qs: [Querystring],
      _auth: [Auth],
      _oauth: [OAuth],
      _multipart: [Multipart],
      _redirect: [Redirect],
      _tunnel: [Tunnel],
      _rp_resolve: [Function (anonymous)],
      _rp_reject: [Function (anonymous)],
      _rp_promise: [Promise [Object]],
      _rp_callbackOrig: undefined,
      callback: [Function (anonymous)],
      _rp_options: [Object],
      headers: [Object],
      setHeader: [Function (anonymous)],
      hasHeader: [Function (anonymous)],
      getHeader: [Function (anonymous)],
      removeHeader: [Function (anonymous)],
      localAddress: undefined,
      pool: {},
      dests: [],
      __isRequestRequest: true,
      _callback: [Function: RP$callback],
      proxy: null,
      tunnel: false,
      setHost: true,
      originalCookieHeader: undefined,
      _disableCookies: true,
      _jar: undefined,
      port: '20041',
      host: 'localhost',
      path: '/session/C00F6564-9225-4157-B38E-1DF094A886D7/orientation',
      _json: true,
      httpModule: [Object],
      agentClass: [Function],
      agent: [Agent],
      _started: true,
      href: 'http://localhost:20041/session/C00F6564-9225-4157-B38E-1DF094A886D7/orientation',
      req: [ClientRequest],
      ntick: true,
      response: [Circular *1],
      originalHost: 'localhost:20041',
      originalHostHeaderName: 'host',
      responseContent: [Circular *1],
      _destdata: true,
      _ended: true,
      _callbackCalled: true,
      [Symbol(kCapture)]: false
    },
    toJSON: [Function: responseToJSON],
    caseless: Caseless { dict: [Object] },
    body: {
      value: [Object],
      sessionId: 'C00F6564-9225-4157-B38E-1DF094A886D7'
    },
    [Symbol(kCapture)]: false,
    [Symbol(kHeaders)]: {
      server: 'WebDriverAgent/1.0',
      'content-type': 'application/json;charset=UTF-8',
      'access-control-allow-origin': '*',
      date: 'Thu, 11 May 2023 15:30:17 GMT',
      'access-control-allow-headers': 'Content-Type, X-Requested-With',
      'accept-ranges': 'bytes',
      'content-length': '385',
      connection: 'close'
    },
    [Symbol(kHeadersCount)]: 16,
    [Symbol(kTrailers)]: null,
    [Symbol(kTrailersCount)]: 0,
    [Symbol(RequestTimeout)]: undefined
  }
}
2023-05-11T15:30:17.691Z FTL/util:lifecycle 7599 [DBF722DB-C6BB-4F72-9828-438DE90795B7] Shutting down due to fatal error with optional error :  StatusCodeError: 404 - {"value":{"error":"unknown command","message":"Unhandled endpoint: /session/C00F6564-9225-4157-B38E-1DF094A886D7/orientation -- http://localhost:20041/ with parameters {\n    wildcards =     (\n        \"session/C00F6564-9225-4157-B38E-1DF094A886D7/orientation\"\n    );\n}","traceback":""},"sessionId":"C00F6564-9225-4157-B38E-1DF094A886D7"}
    at new StatusCodeError (/Users/idmcloud/tools/mcloud-ios/stf/node_modules/request-promise-core/lib/errors.js:32:15)
    at Request.plumbing.callback (/Users/idmcloud/tools/mcloud-ios/stf/node_modules/request-promise-core/lib/plumbing.js:104:33)
    at Request.RP$callback [as _callback] (/Users/idmcloud/tools/mcloud-ios/stf/node_modules/request-promise-core/lib/plumbing.js:46:31)
    at Request.self.callback (/Users/idmcloud/tools/mcloud-ios/stf/node_modules/request/request.js:185:22)
    at Request.emit (node:events:390:28)
    at Request.<anonymous> (/Users/idmcloud/tools/mcloud-ios/stf/node_modules/request/request.js:1154:10)
    at Request.emit (node:events:390:28)
    at IncomingMessage.<anonymous> (/Users/idmcloud/tools/mcloud-ios/stf/node_modules/request/request.js:1076:12)
    at Object.onceWrapper (node:events:509:28)
    at IncomingMessage.emit (node:events:402:35)
    at endReadableNT (node:internal/streams/readable:1340:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) {
  statusCode: 404,
  error: {
    value: {
      error: 'unknown command',
      message: 'Unhandled endpoint: /session/C00F6564-9225-4157-B38E-1DF094A886D7/orientation -- http://localhost:20041/ with parameters {\n' +
        '    wildcards =     (\n' +
        '        "session/C00F6564-9225-4157-B38E-1DF094A886D7/orientation"\n' +
        '    );\n' +
        '}',
      traceback: ''
    },
    sessionId: 'C00F6564-9225-4157-B38E-1DF094A886D7'
  },
  options: {
    method: 'GET',
    uri: 'http://localhost:20041/session/C00F6564-9225-4157-B38E-1DF094A886D7/orientation',
    json: true,
    callback: [Function: RP$callback],
    transform: undefined,
    simple: true,
    resolveWithFullResponse: false,
    transform2xxOnly: false
  },
  response: <ref *1> IncomingMessage {
    _readableState: ReadableState {
      objectMode: false,
      highWaterMark: 16384,
      buffer: BufferList { head: null, tail: null, length: 0 },
      length: 0,
      pipes: [],
      flowing: true,
      ended: true,
      endEmitted: true,
      reading: false,
      constructed: true,
      sync: true,
      needReadable: false,
      emittedReadable: false,
      readableListening: false,
      resumeScheduled: false,
      errorEmitted: false,
      emitClose: true,
      autoDestroy: true,
      destroyed: true,
      errored: null,
      closed: true,
      closeEmitted: true,
      defaultEncoding: 'utf8',
      awaitDrainWriters: null,
      multiAwaitDrain: false,
      readingMore: true,
      dataEmitted: true,
      decoder: null,
      encoding: null,
      [Symbol(kPaused)]: false
    },
    _events: [Object: null prototype] {
      end: [Array],
      close: [Array],
      data: [Function (anonymous)],
      error: [Function (anonymous)]
    },
    _eventsCount: 4,
    _maxListeners: undefined,
    socket: Socket {
      connecting: false,
      _hadError: false,
      _parent: null,
      _host: 'localhost',
      _readableState: [ReadableState],
      _events: [Object: null prototype],
      _eventsCount: 7,
      _maxListeners: undefined,
      _writableState: [WritableState],
      allowHalfOpen: false,
      _sockname: null,
      _pendingData: null,
      _pendingEncoding: '',
      server: null,
      _server: null,
      parser: null,
      _httpMessage: [ClientRequest],
      [Symbol(async_id_symbol)]: 130,
      [Symbol(kHandle)]: [TCP],
      [Symbol(kSetNoDelay)]: false,
      [Symbol(lastWriteQueueSize)]: 0,
      [Symbol(timeout)]: null,
      [Symbol(kBuffer)]: null,
      [Symbol(kBufferCb)]: null,
      [Symbol(kBufferGen)]: null,
      [Symbol(kCapture)]: false,
      [Symbol(kBytesRead)]: 0,
      [Symbol(kBytesWritten)]: 0,
      [Symbol(RequestTimeout)]: undefined
    },
    httpVersionMajor: 1,
    httpVersionMinor: 1,
    httpVersion: '1.1',
    complete: true,
    rawHeaders: [
      'Server',
      'WebDriverAgent/1.0',
      'Content-Type',
      'application/json;charset=UTF-8',
      'Access-Control-Allow-Origin',
      '*',
      'Date',
      'Thu, 11 May 2023 15:30:17 GMT',
      'Access-Control-Allow-Headers',
      'Content-Type, X-Requested-With',
      'Accept-Ranges',
      'bytes',
      'Content-Length',
      '385',
      'Connection',
      'close'
    ],
    rawTrailers: [],
    aborted: false,
    upgrade: false,
    url: '',
    method: null,
    statusCode: 404,
    statusMessage: 'Not Found',
    client: Socket {
      connecting: false,
      _hadError: false,
      _parent: null,
      _host: 'localhost',
      _readableState: [ReadableState],
      _events: [Object: null prototype],
      _eventsCount: 7,
      _maxListeners: undefined,
      _writableState: [WritableState],
      allowHalfOpen: false,
      _sockname: null,
      _pendingData: null,
      _pendingEncoding: '',
      server: null,
      _server: null,
      parser: null,
      _httpMessage: [ClientRequest],
      [Symbol(async_id_symbol)]: 130,
      [Symbol(kHandle)]: [TCP],
      [Symbol(kSetNoDelay)]: false,
      [Symbol(lastWriteQueueSize)]: 0,
      [Symbol(timeout)]: null,
      [Symbol(kBuffer)]: null,
      [Symbol(kBufferCb)]: null,
      [Symbol(kBufferGen)]: null,
      [Symbol(kCapture)]: false,
      [Symbol(kBytesRead)]: 0,
      [Symbol(kBytesWritten)]: 0,
      [Symbol(RequestTimeout)]: undefined
    },
    _consuming: false,
    _dumped: false,
    req: ClientRequest {
      _events: [Object: null prototype],
      _eventsCount: 5,
      _maxListeners: undefined,
      outputData: [],
      outputSize: 0,
      writable: true,
      destroyed: false,
      _last: true,
      chunkedEncoding: false,
      shouldKeepAlive: false,
      maxRequestsOnConnectionReached: false,
      _defaultKeepAlive: true,
      useChunkedEncodingByDefault: false,
      sendDate: false,
      _removedConnection: false,
      _removedContLen: false,
      _removedTE: false,
      _contentLength: 0,
      _hasBody: true,
      _trailer: '',
      finished: true,
      _headerSent: true,
      _closed: false,
      socket: [Socket],
      _header: 'GET /session/C00F6564-9225-4157-B38E-1DF094A886D7/orientation HTTP/1.1\r\n' +
        'host: localhost:20041\r\n' +
        'accept: application/json\r\n' +
        'Connection: close\r\n' +
        '\r\n',
      _keepAliveTimeout: 0,
      _onPendingData: [Function: nop],
      agent: [Agent],
      socketPath: undefined,
      method: 'GET',
      maxHeaderSize: undefined,
      insecureHTTPParser: undefined,
      path: '/session/C00F6564-9225-4157-B38E-1DF094A886D7/orientation',
      _ended: true,
      res: [Circular *1],
      aborted: false,
      timeoutCb: null,
      upgradeOrConnect: false,
      parser: null,
      maxHeadersCount: null,
      reusedSocket: false,
      host: 'localhost',
      protocol: 'http:',
      [Symbol(kCapture)]: false,
      [Symbol(kNeedDrain)]: false,
      [Symbol(corked)]: 0,
      [Symbol(kOutHeaders)]: [Object: null prototype]
    },
    request: Request {
      _events: [Object: null prototype],
      _eventsCount: 5,
      _maxListeners: undefined,
      method: 'GET',
      uri: [Url],
      readable: true,
      writable: true,
      explicitMethod: true,
      _qs: [Querystring],
      _auth: [Auth],
      _oauth: [OAuth],
      _multipart: [Multipart],
      _redirect: [Redirect],
      _tunnel: [Tunnel],
      _rp_resolve: [Function (anonymous)],
      _rp_reject: [Function (anonymous)],
      _rp_promise: [Promise [Object]],
      _rp_callbackOrig: undefined,
      callback: [Function (anonymous)],
      _rp_options: [Object],
      headers: [Object],
      setHeader: [Function (anonymous)],
      hasHeader: [Function (anonymous)],
      getHeader: [Function (anonymous)],
      removeHeader: [Function (anonymous)],
      localAddress: undefined,
      pool: {},
      dests: [],
      __isRequestRequest: true,
      _callback: [Function: RP$callback],
      proxy: null,
      tunnel: false,
      setHost: true,
      originalCookieHeader: undefined,
      _disableCookies: true,
      _jar: undefined,
      port: '20041',
      host: 'localhost',
      path: '/session/C00F6564-9225-4157-B38E-1DF094A886D7/orientation',
      _json: true,
      httpModule: [Object],
      agentClass: [Function],
      agent: [Agent],
      _started: true,
      href: 'http://localhost:20041/session/C00F6564-9225-4157-B38E-1DF094A886D7/orientation',
      req: [ClientRequest],
      ntick: true,
      response: [Circular *1],
      originalHost: 'localhost:20041',
      originalHostHeaderName: 'host',
      responseContent: [Circular *1],
      _destdata: true,
      _ended: true,
      _callbackCalled: true,
      [Symbol(kCapture)]: false
    },
    toJSON: [Function: responseToJSON],
    caseless: Caseless { dict: [Object] },
    body: {
      value: [Object],
      sessionId: 'C00F6564-9225-4157-B38E-1DF094A886D7'
    },
    [Symbol(kCapture)]: false,
    [Symbol(kHeaders)]: {
      server: 'WebDriverAgent/1.0',
      'content-type': 'application/json;charset=UTF-8',
      'access-control-allow-origin': '*',
      date: 'Thu, 11 May 2023 15:30:17 GMT',
      'access-control-allow-headers': 'Content-Type, X-Requested-With',
      'accept-ranges': 'bytes',
      'content-length': '385',
      connection: 'close'
    },
    [Symbol(kHeadersCount)]: 16,
    [Symbol(kTrailers)]: null,
    [Symbol(kTrailersCount)]: 0,
    [Symbol(RequestTimeout)]: undefined
  }
}
2023-05-11 18:30:17.707 WebDriverAgentRunner_tvOS-Runner[7199:442019] Disconnected a client from screenshots broadcast
2023-05-11 18:30:17.707 WebDriverAgentRunner_tvOS-Runner[7199:442019] Disconnected a client from screenshots broadcast
2023-05-11 18:30:19.259 WebDriverAgentRunner_tvOS-Runner[7199:438509] Continuing to run tests in the background with task ID 2
com.facebook.WebDriverAgentRunner.xctrunner: 7199
/Users/idmcloud/tools/mcloud-ios/metaData/Apple_TV.env
Apple_TV (DBF722DB-C6BB-4F72-9828-438DE90795B7)
Starting WDA: Apple_TV, udid: DBF722DB-C6BB-4F72-9828-438DE90795B7, WDA_PORT: 20041, MJPEG_PORT: 20042
2023-05-11 18:30:22.344 WebDriverAgentRunner_tvOS-Runner[8969:443618] Running tests...
2023-05-11 18:30:22.348 WebDriverAgentRunner_tvOS-Runner[8969:443618] Found most recent test bundle at NSBundle </Users/idmcloud/Library/Developer/CoreSimulator/Devices/DBF722DB-C6BB-4F72-9828-438DE90795B7/data/Containers/Bundle/Application/2C83E2B5-31AB-46C8-823F-F431B977530F/WebDriverAgentRunner_tvOS-Runner.app/PlugIns/WebDriverAgentRunner_tvOS.xctest> (not yet loaded)
2023-05-11 18:30:22.348 WebDriverAgentRunner_tvOS-Runner[8969:443618] Synthesizing a test configuration for the test bundle
2023-05-11 18:30:22.349 WebDriverAgentRunner_tvOS-Runner[8969:443618] Running with configuration <XCTestConfiguration: 0x7fc07d707e30>
	                  testBundleURL:file:///Users/idmcloud/Library/Developer/CoreSimulator/Devices/DBF722DB-C6BB-4F72-9828-438DE90795B7/data/Containers/Bundle/Application/2C83E2B5-31AB-46C8-823F-F431B977530F/WebDriverAgentRunner_tvOS-Runner.app/PlugIns/WebDriverAgentRunner_tvOS.xctest/
	         testBundleRelativePath:(null)
	              productModuleName:(null)
	                    testsToSkip:(null)
	                     testsToRun:(null)
	             reportResultsToIDE:no
	               testsDrivenByIDE:no
	              sessionIdentifier:(null)
	      disablePerformanceMetrics:no
	treatMissingBaselinesAsFailures:no
	                baselineFileURL:(null)
	       baselineFileRelativePath:(null)
	          targetApplicationPath:(null)
	      targetApplicationBundleID:(null)
	 testApplicationDependencies:(null)
	 testApplicationUserOverrides:(null)
	     targetApplicationArguments:(null)
	   targetApplicationEnvironment:(null)
	                  reportActivities:YES
	          testsMustRunOnMainThread:no
	            initializeForUITesting:YES
	          initializeForMultiDevice:no
	    shouldExtractMultiDeviceRequirements:no
	     multiDevicePlatformVersionMap:null
	   multiDeviceRequirementsFilePath:(null)
	            traceCollectionEnabled:(null)
	 enablePerformanceTestsDiagnostics:(null)
	 aggregateStatisticsBeforeCrash:(null)
	        automationFrameworkPath:(null)
	                     emitOSLogs:YES
	       systemAttachmentLifetime:0
	         userAttachmentLifetime:0
	   gatherLocalizableStringsData:no
	          testExecutionOrdering:alphabetical
	    randomExecutionOrderingSeed:(null)
	               repetitionPolicy:<XCTRepetitionPolicy: 0x6000025bee40>
	            mode:None
	            maximumIterations:1
	            repeatInNewRunnerProcess:NO

	            testTimeoutsEnabled:NO
	 defaultTestExecutionTimeAllowance:600
	 maximumTestExecutionTimeAllowance:0
	                IDECapabilities:(null)
	         applicationBundleInfos:(null)
WDA not started yet on Apple_TV. waiting 0 sec...
Test Suite 'All tests' started at 2023-05-11 18:30:22.728
Test Suite 'WebDriverAgentRunner_tvOS.xctest' started at 2023-05-11 18:30:22.729
Test Suite 'UITestingUITests' started at 2023-05-11 18:30:22.729
    t =      nans Suite Set Up
Test Case '-[UITestingUITests testRunner]' started.
    t =     0.00s Start Test at 2023-05-11 18:30:22.795
    t =     0.00s Set Up
2023-05-11 18:30:22.797 WebDriverAgentRunner_tvOS-Runner[8969:443618] Built at Jan 17 2023 16:01:57
2023-05-11 18:30:22.821 WebDriverAgentRunner_tvOS-Runner[8969:443618] ServerURLHere->http://192.168.89.189:20041<-ServerURLHere
2023-05-11 18:30:22.822 WebDriverAgentRunner_tvOS-Runner[8969:443632] Using singleton test manager
WDA started successfully on Apple_TV within 1 seconds.
2023-05-11 18:30:23.953 WebDriverAgentRunner_tvOS-Runner[8969:443618] Getting the most recent active application (out of 1 total items)
2023-05-11 18:30:23.953 WebDriverAgentRunner_tvOS-Runner[8969:443618] Cannot retrieve any active applications. Assuming the system application is the active one
    t =     1.17s Find the Application 'com.apple.PineBoard'
    t =     1.17s     Requesting snapshot of accessibility hierarchy for app with pid 24448
Starting iSTF ios-device: DBF722DB-C6BB-4F72-9828-438DE90795B7 device name : Apple_TV
Now using node v17.1.0 (npm v8.3.0)
2023-05-11T15:30:26.535Z INF/db 9376 [DBF722DB-C6BB-4F72-9828-438DE90795B7] Connecting to demo.zebrunner.farm:28015
2023-05-11T15:30:26.542Z INF/device:support:push 9376 [DBF722DB-C6BB-4F72-9828-438DE90795B7] Ios device sending output to "tcp://demo.zebrunner.farm:7270"
2023-05-11T15:30:26.544Z INF/ios-device 9376 [DBF722DB-C6BB-4F72-9828-438DE90795B7] Preparing device options:  {
  serial: 'DBF722DB-C6BB-4F72-9828-438DE90795B7',
  provider: 'mcloud-ios',
  publicIp: 'demo.zebrunner.farm',
  endpoints: {
    sub: [ 'tcp://demo.zebrunner.farm:7250' ],
    push: [ 'tcp://demo.zebrunner.farm:7270' ],
    appDealer: [ 'tcp://demo.zebrunner.farm:7160' ],
    devDealer: [ 'tcp://demo.zebrunner.farm:7260' ]
  },
  groupTimeout: 3600000,
  storageUrl: 'https://demo.zebrunner.farm:443/',
  adbHost: '127.0.0.1',
  adbPort: 5037,
  screenJpegQuality: 30,
  screenPingInterval: 30000,
  screenPort: 7741,
  screenWsUrlPattern: 'wss://demo.zebrunner.farm:443/d/192.168.89.189/DBF722DB-C6BB-4F72-9828-438DE90795B7/7741/',
  connectUrlPattern: '${publicIp}:${publicPort}',
  connectPort: 20042,
  wdaHost: 'localhost',
  wdaPort: 20041,
  vncInitialSize: [ 600, 800 ],
  heartbeatInterval: 10000,
  bootCompleteTimeout: 60000,
  muteMaster: 'never',
  lockRotation: false,
  cleanup: false,
  screenReset: true,
  udidStorage: false,
  iproxy: false,
  deviceName: 'Apple_TV',
  deviceType: 'tvOS',
  host: '192.168.89.189'
}
2023-05-11T15:30:26.885Z INF/device:support:sub 9376 [DBF722DB-C6BB-4F72-9828-438DE90795B7] Receiving input from "tcp://demo.zebrunner.farm:7250"
2023-05-11T15:30:26.886Z INF/device:support:sub 9376 [DBF722DB-C6BB-4F72-9828-438DE90795B7] Subscribing to permanent channel "*ALL"
2023-05-11T15:30:26.889Z INF/device:plugins:solo 9376 [DBF722DB-C6BB-4F72-9828-438DE90795B7] Subscribing to permanent channel "NBr+UpPh/I1as/90nggIPNoZUVw="
2023-05-11T15:30:26.892Z INF/wdaClient 9376 [DBF722DB-C6BB-4F72-9828-438DE90795B7] WdaClient.js initializing...
connecting to WdaMjpeg
2023-05-11 18:30:26.895 WebDriverAgentRunner_tvOS-Runner[8969:443632] Got screenshots broadcast client connection at ::1:60075
2023-05-11T15:30:26.900Z INF/device:info 9376 [DBF722DB-C6BB-4F72-9828-438DE90795B7] device.name: Apple_TV
Connected to WdaMjpeg localhost:20042
```

**Comment by vdelendik** [comment]
ticket should be fixed in `stf:2.4.7-beta2`

**Comment by dhreben** [comment]
Fixed.
